app/models/validations/sales/sale_information_validations.rb

```diff
@@ -15,6 +15,16 @@ def validate_pratical_completion_date_before_saledate(record)
     end
   end
 
+  def validate_exchange_date(record)
+    return unless record.exdate && record.saledate
+
+    record.errors.add(:exdate, I18n.t("validations.sale_information.exdate.must_be_before_saledate")) if record.exdate > record.saledate
+
+    return if (record.saledate.to_date - record.exdate.to_date).to_i / 365 < 1
+
+    record.errors.add(:exdate, I18n.t("validations.sale_information.exdate.must_be_less_than_1_year_from_saledate")) if record.exdate < record.saledate
```

Comment:
```suggestion
    record.errors.add(:exdate, I18n.t("validations.sale_information.exdate.must_be_less_than_1_year_from_saledate"))
```

I can't work out if this is implicit or not from your early return above?